packages/google-cloud-batch/samples/basics/create_with_script_no_mounting.js

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(projectId, region, jobName) {
```

Comment:
Please add jsdoc to the sample, even if it's not included in region tag.
Example:
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/iam/deny/createDenyPolicy.js#L26

Reply:
Done